prometheus.lua

```diff
@@ -115,6 +115,39 @@ function Gauge:set(value, label_values)
   self.prometheus:set(self.name, self.label_names, label_values, value)
 end
 
+
+-- Increase a given gauge by `value`
+--
+-- Args:
+--   value: (number) a value to add to the gauge. Defaults to 1 if skipped.
+--   label_values: an array of label values. Can be nil (i.e. not defined) for
+--     metrics that have no labels.
+function Gauge:inc(value, label_values)
+  local err = self:check_label_values(label_values)
+  if err ~= nil then
+    self.prometheus:log_error(err)
+    return
+  end
+  self.prometheus:inc(self.name, self.label_names, label_values, value or 1)
+end
+
+
+-- Decrease a given gauge by `value`
+--
+-- Args:
+--   value: (number) subtracts the given value from the gauge. Defaults to 1 if skipped.
+--   label_values: an array of label values. Can be nil (i.e. not defined) for
+--     metrics that have no labels.
+function Gauge:dec(value, label_values)
```

Comment:
I am not sure this is strictly necessary if you can call `Gauge:inc` with a negative value, though I don't mind leaving it here for convenience. If you decide to leave it, please make sure to document it in the README as well.

Reply:
Em.. I decide to remove ``Gauge:dec``  which I think is not necessary any more.